templates/settings/payment.php

```diff
@@ -7,47 +7,124 @@
+<div class="dokan-payment-settings-summary">
+    <h2 id="vendor-dashboard-payment-settings-error"></h2>
+    <div class="payment-methods-listing-header">
+        <h2> <?php esc_html_e( 'Payment Methods', 'dokan-lite' ); ?></h2>
+        <div id="vendor-dashboard-payment-settings-toggle-dropdown">
+            <button id="toggle-vendor-payment-method-drop-down" class="dokan-btn dokan-btn-success"> <?php esc_html_e( 'Add Payment Method', 'dokan-lite' ); ?></button>
+            <div id="vendor-payment-method-drop-down-wrapper">
+                <div id="vendor-payment-method-drop-down">
+                    <?php
+                    $profile_methods = array_keys( $profile_info['payment'] );
+                    $unused_methods  = array_diff( $methods, $profile_methods );
 
-    <?php wp_nonce_field( 'dokan_payment_settings_nonce' ); ?>
+                    $mis_match_methods = [ 'stripe', 'moip' ];
+                    $mis_match_methods = array_filter( $mis_match_methods, function ( $key ) use( $profile_methods ) {
+                        return in_array( $key,  $profile_methods);
+                    } );
+
+                    $unused_methods = array_filter( $unused_methods, function ( $key ) use ( $mis_match_methods ) {
+                        $found = false;
+
+                        foreach ( $mis_match_methods as $mis_match_method ) {
+                            $found = $found || ( false !== stripos( $key, $mis_match_method ) );
+                        }
+
+                        return ! $found;
+                    } );
+                    ?>
+                    <ul>
+                    <?php foreach ( $unused_methods as $method_key ) {
+                        $method = dokan_withdraw_get_method( $method_key );
+
+                        if ( ! empty( $method ) ) {
+                            $has_methods_dropdown = true;
+                        } else {
+                            continue;
+                        }
+
+                        ?>
+                        <li>
+                            <a href="<?php echo esc_url(  home_url( "dashboard/settings/payment/manage-" . $method_key ) ); ?>">
+                                <div>
+                                    <img src="<?php echo esc_url( DOKAN_PLUGIN_ASSEST . "/images/dashboard-settings/payment/$method_key.svg" ); ?>" alt="<?php echo esc_attr( $method_key ); ?>" />
+                                    <span> <?php
+                                        //translators: %s: payment method title
+                                        echo sprintf( __( 'Direct to %s', 'dokan-lite' ), esc_html__( $method['title'], 'dokan-lite' ) );
+                                        ?>
+                                    </span>
+                                </div>
+                            </a>
+                        </li>
+                    <?php } ?>
+                    </ul>
+
+                    <?php if ( ! $has_methods_dropdown ) { ?>
```

Comment:
How this check is relevant here? It seems you are printing the following html if there is no unused methods. If so, check if the `$unused_methods` is empty and if not empty then go for the previous block.

Reply:
the if-empty decision is taken after considering the return value of `dokan_withdraw_get_method`.